### PR TITLE
add getSize to youtube-player

### PR DESCRIPTION
**Pull request by rekliner**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
getSize method used in other repo: https://socket.dev/npm/package/@hercilio/ngx-youtube-player#:~:text=getSize
relevant to react-youtube: https://github.com/tjallingt/react-youtube/issues/201#issuecomment-1413159253  
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/youtube-player/youtube-player-tests.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ player.loadVideoById('doesNotExist');
 player.playVideo();
 player.pauseVideo();
 player.setSize(320, 200);
+console.log('player size:', player.getSize());
```
Comment (jurca) [inline types/youtube-player/youtube-player-tests.ts:56]: Please do not rely on `console.log` for testing code as it does not allow for automated verification.

I suggest you try something like this:

```typescript
const playerSize: Promise<sizeType> = player.getSize();
```

File: types/youtube-player/dist/types.d.ts [hunk 2]
```diff
@@ -42,6 +42,11 @@ export interface IframeApiType {
     Player: {new(elementId: string, options: Options): YouTubePlayer};
 }
 
+export interface sizeType {
```
Comment (jurca) [inline types/youtube-player/dist/types.d.ts:45]: Please use naming conventions in line of other identifiers, and prefer context-conveying names. How about `PlayerSize`?

**Review by jurca — CHANGES_REQUESTED** [review]
Thank you for submitting this PR!

I would like to suggest a few tweaks to make this a little better. Thank you for considering them.

File: types/youtube-player/youtube-player-tests.ts (hunk 1 above)
Comment (rekliner) [reply types/youtube-player/youtube-player-tests.ts]: good point, adjusted.

File: types/youtube-player/dist/types.d.ts (hunk 2 above)
Comment (rekliner) [reply types/youtube-player/dist/types.d.ts]: careless camelCase...amended!

**Review by jurca — APPROVED** [review]
Looks good, thanks a lot!

**Comment by jurca** [comment]
@typescript-bot 
Ready to merge

**Pull request merged**